gapic/templates/setup.py.j2

```diff
@@ -94,6 +98,9 @@ setuptools.setup(
     packages=packages,
     python_requires=">=3.7",
     install_requires=dependencies,
+{% if rest_async_io_enabled %}
+    extras_require=extras,
```

Comment:
So if this wasn't there before, why weren't some tests failing? This would suggest a failure in our tests. At the GAPIC level, the test should silently pass if the GAPIC was generated without REST transport or installed without the extra. But at the generator level, we want to ensure that the tests with the extras pass, so how can we be notified if we run a test suite where we expect the extra and the associated tests, but it turns out the extra wasn't installed?

Reply:
Tests weren't failing because we're not using the `async_rest` extra to install the dependencies. We've explicitly defined each required dependency within the constraints file.